README.md

```diff
@@ -46,168 +49,171 @@ With their functions being:
+volatility: | 
+`bbands`      |bollinger bands ([source 1](https://en.wikipedia.org/wiki/Bollinger_Bands), [source 2](https://www.investopedia.com/terms/b/bollingerbands.asp))
+volume: | 
+`ad`          |chaikin accumulation/distribution line values ([source 1](https://en.wikipedia.org/wiki/Accumulation/distribution_index), [source 2](https://www.investopedia.com/terms/a/accumulationdistribution.asp))
+`obv`         |on balance volume ([source 1](https://en.wikipedia.org/wiki/On-balance_volume), [source 2](https://www.investopedia.com/terms/o/onbalancevolume.asp))
 
 ### [Due Diligence](due_diligence/README.md) <a name="Due_Diligence"></a>
-```
-news          latest news of the company [Finviz]
-red           gets due diligence from another user's post [Reddit]
-analyst       analyst prices and ratings of the company [Finviz]
-rating        rating of the company from strong sell to strong buy [FMP]
-pt            price targets over time [Business Insider]
-est           quarter and year analysts earnings estimates [Business Insider]
-ins           insider activity over time [Business Insider]
-insider       insider trading of the company [Finviz]
-sec           SEC filings [Market Watch]
-short         short interest [Quandl]
-warnings      company warnings according to Sean Seah book [Market Watch]
-```
+Command|Explanation
+------ | --------
+`news`          |latest news of the company ([Finviz](https://finviz.com/))
+`red`           |gets due diligence from another user's post ([Reddit](https://reddit.com))
+`analyst`       |analyst prices and ratings of the company ([Finviz](https://finviz.com/))
+`rating`        |rating of the company from strong sell to strong buy [FMP]
+`pt`            |price targets over time ([Business Insider](https://www.businessinsider.com/))
+`est`           |quarter and year analysts earnings estimates ([Business Insider](https://www.businessinsider.com/))
+`ins`           |insider activity over time ([Business Insider](https://www.businessinsider.com/))
+`insider`       |insider trading of the company ([Finviz](https://finviz.com/))
+`sec`           |SEC filings ([MarketWatch](https://www.marketwatch.com/))
+`short`         |short interest ([Quandl](https://www.quandl.com/))
+`warnings`      |company warnings according to Sean Seah book ([MarketWatch](https://www.marketwatch.com/))
+
 
 ### [Prediction Techniques](prediction_techniques/README.md) <a name="Prediction_Techniques"></a>
-```
-sma         simple moving average
-knn         k-Nearest Neighbors
-linear      linear regression (polynomial 1)
-quadratic   quadratic regression (polynomial 2)
-cubic       cubic regression (polynomial 3)
-regression  regression (other polynomial)
-arima       autoregressive integrated moving average
-prophet     Facebook's prophet prediction
-mlp         MultiLayer Perceptron
-rnn         Recurrent Neural Network
-lstm        Long-Short Term Memory
-```
+Command|Algorithm
```

Comment:
I would use: Command|Technique|Source, instead.

Love the source addition! Would also call it Wiki, Investopedia instead.